apps/src/templates/sectionProgress/standards/sectionStandardsProgressRedux.js

```diff
@@ -25,8 +23,25 @@ export default function sectionStandardsProgress(state = initialState, action) {
   return state;
 }
 
-export function getUnpluggedLessonsForScript(script) {
-  return unpluggedLessonList;
+export function getUnpluggedLessonsForScript(state) {
+  const stages =
+    state.sectionProgress.scriptDataByScript[state.scriptSelection.scriptId]
+      .stages;
```

Comment:
do we need to be checking that these exist before chaining them? ie, are we sure that `state.sectionProgress.scriptDataByScript[state.scriptSelection.scriptId]` will always exist?